cfpb/consumerfinance.gov
# P4C: form accessibility improvements

Use appropriate form elements to improve accessibility in P4C grad path tool.


## Changes

- Wrap radio button groups in fieldsets with legends
- Convert headings to labels where needed


## How to test this PR

1. Check out branch and run gulp
2. Using WAVE tool, verify that the updated screens (`school search`, `affording your loan`, `action plan`, and `save & finish`) do not report any "missing fieldset", "orphaned form label", or other form-related errors


## Screenshots

### Before
<img width="680" alt="Screen Shot 2021-04-06 at 2 00 41 PM" src="https://user-images.githubusercontent.com/778171/113777941-9a453680-96e0-11eb-9ec7-05dabbecd1ac.png">


### After
<img width="655" alt="Screen Shot 2021-04-06 at 2 01 12 PM" src="https://user-images.githubusercontent.com/778171/113777952-9dd8bd80-96e0-11eb-8acc-bac2e9f52220.png">





## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance



#### Accessibility
